### ISSUE TITLE
SSM logging expects script binary to be available

### ISSUE THREAD

**Issue by chris-vest** [bottlerocket-os/bottlerocket-control-container#24]
**What I'd like:**

To have `script` installed such that the SSM logging works.

Currently I get this message ending up in the log bucket:

```
unable to generate log data: Failed to generate transcript with the following errors:
exec: "script": executable file not found in $PATH:
exec: "script": executable file not found in $PATH:
```

For example, I run:

```aws ssm start-session --target i-xyz```

I am now connected to the instance. I should expect to see logs in the log bucket which is configured as part of SSM, however I only see a `stderr` file with the logs shown above.

**Any alternatives you've considered:**

Not sure there is another option, I assume it depends on what the SSM agent expects is available on the node where it's running.

**Comment by etungsten** [comment]
Hi @chris-vest, thanks for bringing this up. I've been able to reproduce the problem. We'll take a closer look at what's going on.

In the meantime, since this issue is specific to the SSM agent running in the [Bottlerocket control host-container](https://github.com/bottlerocket-os/bottlerocket-control-container), I'm gonna transfer this issue to that repository.

**Comment by jpculp** [comment]
Hi @chris-vest. I've merged a PR that should resolve the problem and it will be part of our next release of Bottlerocket. If you would like to get a head start, you can build the container from develop and set the source via user-data ([detailed here](https://github.com/bottlerocket-os/bottlerocket#custom-host-containers)). I'll leave this open until the next release.

**Comment by jpculp** [comment]
As of today, Bottlerocket [v1.7.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.7.0) has been released, which uses [v0.6.0](https://github.com/bottlerocket-os/bottlerocket-control-container/releases/tag/v0.6.0) of the control container that contains `script`. Don't hesitate to reach back out if you run into any further issues!
